haskell-lsp-types/src/Language/Haskell/LSP/Types/Message.hs

```diff
@@ -50,8 +50,8 @@ import GHC.Generics
 
 -- | Map a method to the message payload type
 type family MessageParams (m :: Method p t) :: Type where
-  --   RequestMessage <method> <params> <response>
-  -- | NotificationMessage <method> <params>
+  --   RequestMessage method params response
+  --   | NotificationMessage <method> <params>
```

Comment:
Unrelated to this PR, but this comment should be removed as it is a leftover from an earlier version of the patch.